macros/sfdc_formula_view.sql

```diff
@@ -7,46 +7,23 @@
     )
 }}
 
-/*
-    The below sets the old_formula_fields variable to the results of the get_column_values results which queries the field column from the fivetran_formula table.
-    The logic here is that the variable will be a list of all current salesforce formula field names. This list is then used within the dbt_utils.star operation to exclude them.
-    This allows users with the Fivetran legacy Salesforce fields to ignore them and be replaced by the new fields.
-*/
-
-{% if full_statement_version %}
-{% if using_quoted_identifiers %}
-{%- set table_results = dbt_utils.get_column_values(table=source(source_name, 'fivetran_formula_model'), 
-                                                    column='"MODEL"' if target.type in ('snowflake') else '"model"' if target.type in ('postgres', 'redshift', 'snowflake') else '`model`', 
-                                                    where=("\"OBJECT\" = '" if target.type in ('snowflake') else "\"object\" = '" if target.type in ('postgres', 'redshift') else "`object` = '") ~ source_table ~ "'") -%}
-
-{% else %}
-{%- set table_results = dbt_utils.get_column_values(table=source(source_name, 'fivetran_formula_model'), column='model', where="object = '" ~ source_table ~ "'") -%}
-
-{% endif %}
-
-{{ table_results[0] }}
+-- Raise a warning if users are trying to use full_statement_version=false. We are keeping the variable in the macro, however, since we don't want errors if they previously set it to true.
+{% if not full_statement_version %}
+    {{ exceptions.warn("\ERROR: The full_statement_version=false, reserved_table_name, and fields_to_include parameters are no longer supported. Please update your " ~ this.identifier|upper ~ " model to remove these parameters.\n") }}
+    * See full model error in log.
```

Comment:
In BigQuery this error is a bit confusion.

![image](https://github.com/fivetran/dbt_salesforce_formula_utils/assets/74217849/b840b240-231b-4b0e-927e-313074265540)

What if we do this instead?
```suggestion
    See_full_model_error_in_log
```

Which provides the following error.
![image](https://github.com/fivetran/dbt_salesforce_formula_utils/assets/74217849/b9ed6b68-9f0b-4b8e-b6c9-903fd15b529d)

Reply:
I see. That works for me!